aldryn_newsblog/south_migrations/0027_rename_plugins_to_include_appname.py

```diff
@@ -0,0 +1,320 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
```

Comment:
I read in the south docs once, that data and schema migrations should not be done in the same file. I'm not sure why though and if it is really relevant at all.